### ISSUE TITLE
Incorrectly detects bash processes as scrcpy-console

### ISSUE THREAD

**Issue by solomongrundy6** [nokyan/resources#112]
When installing scrcpy, it adds two .desktop files to /usr/share/applications, scrcpy.desktop and scrcpy-console.desktop.  scrcpy.desktop launches the program by just calling 'scrcpy', but scrcpy-console.desktop launches:

```
/bin/bash --norc --noprofile -i -c "\"\\$SHELL\" -i -c scrcpy || read -p 'Press Enter to quit...'"
```

This causes Resources to show scrcpy-console in the applications list when it's not running - it's detecting every bash process as scrcpy-console.  There are a couple ways I can workaround this specifically for scrcpy, but this is probably something that could come up with other .desktop files for people in the future so I figured I'd report it as an issue.  Resources is still properly showing my terminal emulator (kitty) separately in the applications list, so it's not interfering with that.

**Comment by nokyan** [comment]
Hi, thanks for reporting the issue.
I'll look into it, I'll probably blacklist bash from being recognized as an application.

**Comment by nokyan** [comment]
Closing this because it has been resolved with Resources 1.3 (still not on Flathub unfortunately). If there's still an issue related to this, don't hesitate to reopen :)
